editor/import/resource_importer_scene.h

```diff
@@ -153,7 +153,7 @@ VARIANT_ENUM_CAST(EditorScenePostImportPlugin::InternalImportCategory)
 class ResourceImporterScene : public ResourceImporter {
 	GDCLASS(ResourceImporterScene, ResourceImporter);
 
-	Set<Ref<EditorSceneFormatImporter>> importers;
+	mutable Vector<Ref<EditorSceneFormatImporter>> importers;
```

Comment:
What does `mutable` do in this context?

Reply:
I don't know, all the other variables had it. Ain't going to need it. So removing.

Reply:
Well there must be a reason why the other ones have it, so someone needs to find out why :)